nav_msgs/Path   waypoints
bool            publish_path
bool            publish_trajectory
bool            plan_path
bool            plan_trajectory
---
trajectory_msgs/MultiDOFJointTrajectory   trajectory
nav_msgs/Path                             path
float64                                   path_length